test/evergreen.yml

```diff
@@ -647,8 +647,9 @@ variables:
 #########################################################################################
 # The following stress tests are configured to run for six hours via the "-t 360"
 # argument to format.sh: format-stress-test, format-stress-sanitizer-test, and
-# race-condition-stress-sanitizer-test. The smoke and recovery tests run in a loop,
-# with the number of runs adjusted to provide aproximately six hours of testing.
+# race-condition-stress-sanitizer-test. The recovery tests run in a loop, with
+# the number of runs adjusted to provide aproximately six hours of testing.  There
+# is just one much shorter run of the smoke tests
```

Comment:
Missing a period here. Also, we can simply say that the smoke tests are run once (the "much shorter run" is confusing).

Reply:
Fixed.

Reply:
I created WT-8359 for the follow-on work.